tests/Feature/CustomTourViewerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\CustomTour;
+use Aic\Hub\Foundation\Testing\FeatureTestCase as BaseTestCase;
+
+class CustomTourViewerTest extends BaseTestCase
+{
+    protected $seed = true;
+
+    /**
+     * A test to check whether a randomly selected custom tour's title renders
+     * on the viewer page.
+     */
+    public function test_custom_tour_title_renders(): void
+    {
+        $randomCustomTour = CustomTour::inRandomOrder()->first();
+
+        $id = $randomCustomTour ? $randomCustomTour->id : null;
+
+        $response = $this->get(route('custom-tours.show', $id));
+
+        $tourJson = json_decode($randomCustomTour->tour_json, true);
+
+        $response->assertSee($tourJson['title']);
+    }
+
+    /**
+     * A test to check whether a randomly selected custom tour's description renders
+     * on the viewer page.
+     */
+    public function test_custom_tour_description_renders(): void
```

Comment:
As well as checking for the positive case, check that no extra markup is introduced if a description is not provided. I.e. another method such as `test_custom_tour_no_extra_markup_renders()` (which can be extended in future as we add other optional content such as descriptions per artwork)

Reply:
Thanks, I've added in an additional test for this.